cuebot/pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[build-system]
+requires = [
+    "hatchling",
+    "versioningit",
+    "grpcio>=1.62.0",
+    "grpcio-tools>=1.62.0",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "opencue_cuebot"
+dynamic = ["version"]
+dependencies = [
+    "grpcio>=1.62.0",
+    "grpcio-tools>=1.62.0",
+]
+
+[tool.hatch.version]
+source = "versioningit"
```

Comment:
Can you explain better on the PR description how this versioningit works and what will be the impact on how we version the project?

Reply:
Have added it to the PR description